ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_latin.txt

```diff
@@ -296,6 +311,11 @@
 		ethnicities = {
 			10 = mediterranean
 		}
+		
+		mercenary_names = {
+			{ name = "mercenary_company_legion" coat_of_arms = "e_roman_empire" }
```

Comment:
Legions shouldn't really be mercenaries. When I get to converting armies, I intend to convert Imperator Legions (the 2.0 professional armies, not just Roman legions) to men-at-arms. Vanilla seems to agree with me.
![image](https://user-images.githubusercontent.com/29546927/102080719-3801b680-3e0f-11eb-9b81-0779327b6caa.png)

Reply:
I meant it to be a mercenary army specific to the Latin culture. Perhaps _Legio XXXII Mercenaria_ would be a better localisation, which I'll add along with the _fanē_ and the _af_

Reply:
I think I have an idea for a better mercenaries localization: _Numerus_
> Numeri were allied native (or "barbarian") units from outside the Empire who fought alongside the regular forces on a mercenary basis. These were led by their own aristocrats and equipped in traditional fashion. Numbers fluctuated according to circumstances and are largely unknown.

https://en.wikipedia.org/wiki/Numerus_(Roman_military_unit)